<launch>
    <node name="task4_node" pkg="competition_tasks" type="task4" output="screen">
        <param name="task4_medical_x" value="0" type="double"/>
		<param name="task4_medical_y" value="0" type="double"/>
		<param name="task4_medical_z" value="0.15" type="double"/>
        <param name="task4_z" value="1" type="double"/>
		<param name="task4_person_x" value="0" type="double"/>
		<param name="task4_person_y" value="0" type="double"/>
		<param name="task4_place_x" value="1.5" type="double"/>
		<param name="task4_place_y" value="1.5" type="double"/>
        <param name="task4_place_z" value="0.15" type="double"/>
	</node>
</launch>